nisserino/cmd_loop.py

```diff
@@ -0,0 +1,101 @@
+import cmd
+from background import *
+
+daycare = Dog_daycare("Vacker Tass", "Nisse")
+
+
+class Daycare_loop(cmd.Cmd):
+    into = "Welcome to the daycare"
+    prompt = "(Daycare): "
+    file = None
+
+    def do_show_dogs(self, arg):
+        'Show the names of all dogs in the Day-care!'
+        try:
+            daycare.all_dogs()
+        except Exception as e:
+            print(f"Error: {e}")
+
+    def do_add_dog(self, arg):  # Fix strip in some sense
+        'Add a dog: add_dog name,age,owner'
+        try:
+            name, age, owner = arg.split(",")
+            daycare.add_dog(name.lower().strip(), int(age), owner.lower().strip())
```

Comment:
Jag hade lagt detta i en add_dog() funktion så cmd koden innehåller en one liner, samma som https://docs.python.org/3/library/cmd.html#cmd-example

Reply:
```suggestion
            daycare.add_dog(name.lower().strip(), int(age), owner.lower().strip())
```
**Lade till**
def fix_input(arg):
    try:
        return int(arg)
    except ValueError:
        return arg.lower().strip()


def parse(arg):
    try:
        return tuple(map(fix_input, arg.split(",")))
    except Exception as e:
        print(f"Error: {e}")

Vilekt gör att i add dog ser ut som:

    def do_add_dog(self, arg):
        'Add a dog: add_dog name,age,owner'
        daycare.add_dog(*parse(arg))